src/smc-util/misc.coffee

```diff
@@ -2,7 +2,7 @@
 #
 # SageMathCloud: A collaborative web-based interface to Sage, IPython, LaTeX and the Terminal.
 #
-#    Copyright (C) 2014, William Stein
```

Comment:
yep -- we should do this.  I made a ticket for doing this everywhere: https://github.com/sagemathinc/smc/issues/1095